Sieve of Eratosthenes

```diff
@@ -0,0 +1,40 @@
+#include <bits/stdc++.h> 
+using namespace std; 
+  
+void SieveOfEratosthenes(int n) 
+{ 
+    // Create a boolean array "prime[0..n]" and initialize 
+    // all entries it as true. A value in prime[i] will 
+    // finally be false if i is Not a prime, else true. 
+    bool prime[n+1]; 
```

Comment:
VLAs are not part of the C++ standard.
I'm not sure how you compiled this.
Please add the compilation information in the PR's description.

`std::vector` would be a better choice here.